### PR TITLE
Yaw issue140&141 event with image

### LINKED ISSUE #140: Improve Create New Event
- Endpoint URI: */api/events*
- Http Method: *POST*

- API Logic
    - upon Event Creation should upload an Event Image

- Progress
    - [x] development
    - [x] integration Test

### LINKED ISSUE #141: Improve Update Event
- Endpoint URI: */api/events*
- Http Method: *PUT*

- API Logic
    - upon Update Event should update an Event Image (If new Image upload)

- Progress
    - [x] development
    - [x] integration Test

### LINKED ISSUE #145: Test for Event API with Event Image
Create Event and Update Event with Image Upload Integration Test

- Progress
    - [x] Create Event Integration Test
    - [x] Update Event Integration Test

### PR DESCRIPTION
**Pull request by HowardYaw**
resolved #140 
resolved #141 
resolved #145

### CODE REVIEW

File: src/main/java/com/thirdcc/webapp/service/dto/EventDTO.java [hunk 1]
```diff
@@ -32,6 +32,10 @@
 
     private EventStatus status;
 
+    private Long imageStorageId;
```
Comment (CarlosSia) [inline src/main/java/com/thirdcc/webapp/service/dto/EventDTO.java:35]: the imageStorageId is not in imageStorageDTO?

Comment (CarlosSia) [inline src/main/java/com/thirdcc/webapp/service/dto/EventDTO.java:36]: the imageStorageId looks like redundant with imageStorageDTO

Comment (HowardYaw) [reply src/main/java/com/thirdcc/webapp/service/dto/EventDTO.java:36]: @CarlosSia 
ImageStorageId will be save into DB
ImageStorageDTO is used when we response the EventDTO with Image Info

**Review by CarlosSia — DISMISSED** [review]

**Comment by HowardYaw** [comment]
Decided to convert Request Parameter to DTO with multipartfile field
and use `@ModelAttribute`

**Review by CarlosSia — APPROVED** [review]

**Review by LUXIANZE — APPROVED** [review]

**Pull request merged**